pkg/scheduler/internal/cache/cache_test.go

```diff
@@ -1090,7 +1090,11 @@ func TestNodeOperators(t *testing.T) {
 			if !found {
 				t.Errorf("Failed to find node %v in internalcache.", node.Name)
 			}
-			if cache.nodeTree.numNodes != 1 || cache.nodeTree.next() != node.Name {
+			nodesList, err := cache.nodeTree.list()
+			if err != nil {
+				t.Fatalf(err.Error())
```

Comment:
> Re https://github.com/kubernetes/kubernetes/pull/93473#discussion_r463735426

Here is one.

Reply:
ah ok, I didn't get what you meant sorry, I changed this to t.Fatal(err)